README.md

```diff
@@ -245,77 +245,64 @@ TODO: describe more precisely each file
+- `elims/`: Contains proofs of elimination principles.
+  - `CofPushoutSection`: Given a span `s`, in which one of the maps has a left-inverse, and a map `h : Pushout s → D`, proves an elimination principle for `Cofiber h`.
+  - `Lemmas`: Contains technical lemmas about commutative squares over commutative squares.
+  - `SuspSmash`: Gives an elimination principle for `Susp (X ∧ Y)`, the suspension of the smash product.
+- `EM1HSpace`: Defines the `HSpaceStructure` on the Eilenberg–MacLane space `⊙EM₁ G` for an abelian group `G`.
+- `EilenbergMacLane`: Defines the Eilenberg–MacLane spaces `⊙EM G n`, proves that `⊙Ω (⊙EM G (S n))` is pointedly equivalent to `⊙EM G n` for each `n` and that their homotopy groups are as required. See *Eilenberg-MacLane Spaces in Homotopy Type Theory* by Dan Licata and Eric Finster.
+- `EilenbergMacLane1`: Proves that the fundamental group of the Eilenberg–MacLane space `⊙EM₁ G` (which is defined as a HIT) is in fact `G`.
+- `FiberOfWedgeToProduct`: Let `X` of `Y` be two types with basepoints `x₀` and `y₀`. This contains a proof that the fiber of the induced map `X ∨ Y → X × Y` over a point `(x , y)` is equivalent to the join `(x₀ == x) * (y₀ == y)`.
+- `FinWedge`: Contains helper functions and lemmas for dealing with wedges indexed over `Fin I` for some `I : ℕ`.
+- `Freudenthal`: Proves the Freudenthal suspension theorem.
+- `GroupSetsRepresentCovers`: Let `X` be a 0-connected type. This file gives an equivalence between coverings of `X` and `πS 0 X`-sets (where `πS 0 X` is the fundamental group of `X`).
+- `HSpace`: Contains definition(s) of H-spaces and some basic lemmas.
+- `Hopf`: Proves that the total space of the Hopf fibration is `S³`.
+- `HopfConstruction`: Given a 0-connected H-space `X`, constructs a fibration `H` on `Susp A` with total space equivalent to the join `X * X`.
+- `HopfJunior`: Contains `HopfJunior : S¹ → Type₀`, a fibration with fibers equivalent to `Bool` (a.k.a. the 0-sphere) and a proof that its total space is (equivalent to) `S¹`.
+- `IterSuspensionStable`: Contains a reformulation of the Freudenthal suspension theorem.
+- `JoinAssoc3x3`: Gives an equivalence between the joins `(A * B) * C` and `A * (B * C)`. The proof uses the 3x3-lemma.
+- `JoinAssocCubical`: Gives an equivalence between the joins `(A * B) * C` and `A * (B * C)`. The proof involves squares and cubes.
+- `JoinComm`: Gives an equivalence between the joins `A * B` and `B * A`.
+- `JoinSusp`: Contains equivalences `Bool * A ≃ Susp A`, `Susp A * B ≃ Susp (A * B)` and `⊙Sphere m ⊙* X ⊙≃ ⊙Susp^ (S m) X` ((m+1)-fold suspension is equivalent to joining with an m-sphere).
+- `LoopSpaceCircle`: Proves that the fundamental group of the circle is equivalent to the integers.
+- `ModalWedgeExtension`: Lemmas about modalities and the function `X ∨ Y → X × Y` for pointed types `X` and `Y`.
```

Comment:
FYI: This will play the same role as `WedgeExtension` in the generalized Blakers-Massey theorem.